### DISCUSSION TITLE
Supabase pricing structure

### DISCUSSION THREAD

**Discussion by jbiddulph** [supabase/supabase#28224] [General]
I don't get it... ok so the $10 extra was very small after I signed up for PRO plan at $25 per month so it actually costs me $35 per month?!

each new project I create seems to be an additional $10?!

Can I not just have 1 organisation and 1 project but different database names or if on same database prefixing the model names?

how can I get the most out of my PRO account without spending $10 extra for each new project?? seems a bit of a rip off and not £25 per month at all

**Comment by GaryAustin1** [comment]
Just to clarify, the $25 includes your first project as there is a $10 credit to cover that.  But yes each addition instance/project is $10 as you are spinning up a dedicated CPU instance on AWS that runs all the time.

You can certainly use schemas in a database to separate out stuff, but each project will have 1 set of auth users.
